game_board.py

```diff
@@ -1,126 +1,64 @@
 import pyglet
+import resources
+from pyglet.window import mouse
 
+path = '/Users/siczek/Python/final_game/git_repos/resources/'
 
-class Cell:
+class Cell(pyglet.sprite.Sprite):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.is_clicked = False
```

Comment:
As discussed (and honestly just for the record since we agreed not to implement it this way): we could have a separate `Clickable` class with `self.is_clicked`, `left`, `right` etc. and use multiple inheritance.

Reply:
Yes we did, but it will be part of the basic game logic